src/app/ubs/ubs/components/ubs-order-details/ubs-order-certificate/ubs-order-certificate.component.ts

```diff
@@ -396,6 +396,12 @@ export class UbsOrderCertificateComponent implements OnInit, OnDestroy {
       this.clickOnNo = false;
     }
   }
+  public showActivateCetificate([i]): boolean {
+    if (this.certificates.expirationDates[i] && this.certSize && !this.certificates.failed[i] && this.certificateSum < this.showTotal) {
```

Comment:
don't use if operator, you can also use additional constants to decompose the condition here

const isCertificateExpired = !!this.certificates.expirationDates[i];
const isCertSize = !!this.certSize;
const isCertNotFailed = !this.certificates.failed[i];
const isShowTotal = this.certificateSum < this.showTotal;

return isCertificateExpired && isCertSize && isCertNotFailed && isShowTotal;

Reply:
Added changes according to your comment.